src/json_converter.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass, field
+import pandas as pd
+
+from src.data.types import Airport
+
+
+@dataclass
+class DataConverter:
+    raw_data: dict[str, list[Airport]]
+    airports_df: pd.DataFrame = field(init=False)
+    runways: pd.DataFrame = field(init=False)
+    flights: pd.DataFrame = field(init=False)
+    air_quality: pd.DataFrame = field(init=False)
+
+    def __post_init__(self) -> None:
+        self.airports_df = pd.DataFrame(self.raw_data["airports"])
+        self.flights = self.airports_df[["flights"]]
+        self.runways = self.airports_df[["runways"]]
+        self.air_quality = self.airports_df[["air_quality"]]
+
+        self.airports_df = self.airports_df.drop(["flights", "runways"], axis=1)
+
+    def save_runways_to_csv(self, filename: str) -> None:
+        self.runways.to_csv(
+            filename + "./runways.csv", sep=",", index=False, encoding="utf-8"
+        )
```

Comment:
These line breaks seem odd to me as most of the code isn't formatted in the same way - perhaps revert these?